static/responses/json/3/action/package_search?sort=id+asc&start=0&rows=100

```diff
@@ -171,6 +167,168 @@
+            "url": "<!--#echo var="mock_absolute_root_url" -->mock-third-party/example-dataset-2/benchmark.pdf",
+            "cache_last_updated": null,
+            "package_id": "a18d2811-13b0-4838-8bfb-5793433317b0",
+            "name": null,
+            "state": "active",
+            "mimetype_inner": null,
+            "last_modified": null,
+            "position": 1,
+            "revision_id": "7ba78f78-4fa2-4c1c-b889-419260c6f0b1",
+            "url_type": null,
+            "id": "9c32745b-d611-4d8c-bec0-4615fb9464d8",
+            "resource_type": null,
+            "size": null
+          },
+          {
+            "mimetype": null,
+            "cache_url": null,
+            "description": "Example Dataset #2 - summary of category scores (XLS)",
+            "format": "XLS",
+            "url": "<!--#echo var="mock_absolute_root_url" -->mock-third-party/example-dataset-2/all-categories-summary.xls",
+            "cache_last_updated": null,
+            "package_id": "a18d2811-13b0-4838-8bfb-5793433317b0",
+            "name": null,
+            "state": "active",
+            "mimetype_inner": null,
+            "last_modified": null,
+            "position": 2,
+            "revision_id": "7ba78f78-4fa2-4c1c-b889-419260c6f0b1",
+            "url_type": null,
+            "id": "67eefa14-86de-4a3f-8fe3-3804ff08f8cf",
+            "resource_type": null,
+            "size": null
+          },
+          {
+            "mimetype": null,
+            "cache_url": null,
+            "description": "Example Dataset #2 - summary of category scores (CSV)",
+            "format": "CSV",
+            "url": "<!--#echo var="mock_absolute_root_url" -->mock-third-party/example-dataset-2/all-categories-summary.csv",
+            "cache_last_updated": null,
+            "package_id": "a18d2811-13b0-4838-8bfb-5793433317b0",
+            "name": null,
+            "state": "active",
+            "mimetype_inner": null,
+            "last_modified": null,
+            "position": 3,
+            "revision_id": "7ba78f78-4fa2-4c1c-b889-419260c6f0b1",
+            "url_type": null,
+            "id": "27ec6d83-7f13-4daf-ba90-227f4271d7f7",
+            "resource_type": null,
+            "size": null
+          }
+        ],
+        "num_resources": 4,
+        "tags": [
+          {
+            "vocabulary_id": null,
+            "state": "active",
+            "display_name": "Example Data 2",
+            "id": "3a0b24bf-52a3-4b38-888c-e5edaa4df5dg",
+            "name": "example-data"
+          }
+        ],
+        "precision": "percentages rounded to whole integer",
+        "groups": [],
+        "creator_user_id": null,
+        "relationships_as_subject": [],
+        "geographic_coverage": "000000: ",
+        "organization": {
+          "description": "This is an example organization publishing only example datasets.",
+          "created": "2012-06-27T14:48:40.244951",
+          "title": "Test Publisher #1",
```

Comment:
Above, this is called `Test Publisher #1` without the hash.

Reply:
I will set it to `example-publisher-1`, as our CKAN theme replaces organization with publisher